### PR TITLE
[Update] COMET to CDP4-COMET and update logo's; fixes #400

### LINKED ISSUE #400: [Update] logo to CDP4-COMET in readme and application
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-WEB
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
[Update] logo to CDP4-COMET in readme and application

### Steps to Reproduce
NA

### Logs

  - [x] There are no logs regarding the issue
  
### System Configuration

- COMET version:
  - [x] COMET-WEB: 1.1.0

### PR DESCRIPTION
**Pull request by samatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
  - update logo in common and app
  - update COMET to CDP4-COMET

### CODE REVIEW

File: COMETwebapp/Components/Shared/About.razor [hunk 1]
```diff
@@ -33,7 +33,7 @@
     <div class="col-md-12">
         <div class="card">
             <div class="card-body">
-                <p>COMET WEB Community Editition (CE) is the RHEA Group open source Concurrent Design web based application compliant with ECSS-E-TM-10-25 Annex A and Annex C</p>
+                <p>CDP4-COMET WEB Community Editition (CE) is the RHEA Group open source Concurrent Design web based application compliant with ECSS-E-TM-10-25 Annex A and Annex C</p>
```
Comment (nabilatrhea) [inline COMETwebapp/Components/Shared/About.razor:36]: small mistake (Editition) to Edition

**Review by nabilatrhea — CHANGES_REQUESTED** [review]
Name also should be changed in styles.css file

**Review by antoineatrhea — APPROVED** [review]
When typo is fixed

**Review by nabilatrhea — APPROVED** [review]

**Comment by antoineatrhea** [comment]
@samatrhea  can you also provide the favicon please?

**Pull request closed without merging**